docs/integrations/sources/iterable.md

```diff
@@ -48,7 +48,7 @@ The Iterable connector should not run into Iterable API limitations under normal
 ### Requirements
 
 * Iterable Account
-* Iterable API Key
+* Iterable API Key with `standard` permissions. See https://support.iterable.com/hc/en-us/articles/360043464871-API-Keys- for more details.
```

Comment:
@htrueman shouldn't this be read-only like detailed by the user in the linked issue?

Reply:
Hm, but he told that he used `standard` permissions to make it work. See https://github.com/airbytehq/airbyte/issues/7735#issuecomment-978882344

Reply:
nevermind I am blind. you are correct - ignore me!